cql2pgjson/src/test/resources/templates/db_scripts/foreignKey.json

```diff
@@ -57,10 +57,16 @@
           "fieldName": "tablebId",
           "targetTable": "tableb"
         }
+      ],
+      "index": [
+
+        {
+          "fieldName": "cindex",
+          "tOps": "ADD",
+          "caseSensitive": true,
+          "removeAccents": true
+        }
```

Comment:
Removing this new "index" for "cindex" does not make the tests fail.
Either remove this new "index" section or add a test that uses it.